tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+import configparser
+import os
+import pytest
+
+
+@pytest.fixture
+def config_file():
```

Comment:
I think this shouldn't be a fixture, but part of test_get_config_settings instead. This shouldn't be the prefered way to change settings during the tests.

Reply:
Ok I removed the pytest fixture